### DISCUSSION TITLE
Welcome to GoWired-Project Discussions!

### DISCUSSION THREAD

**Discussion by feanor-anglin** [GoWired/GoWired-Project#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by szurr** [comment]
Hi i look forward at this project. I am in the process of making my home smarter with use of wired solutions. At the moment I have blinds and thermostats via USB gate and planning much more. I was wondering is there a plans for pjon support. Mysensors should release it soon officialy.

**Reply by feanor-anglin** [reply]
Hi @szurr, very nice to see your comment just after I've set up the new discussion feature.

Are you using MySensors USB gateway? How does it work for you, any problems? I'm asking since from ours experience, Serial Gateway was far less stable than Ethernet one.

About PJON, I'm not really familiar with it, but as far as I understand its documentation, it can work through Serial and therefore also RS485. So if PJON was integrated into MySensors, it would be almost automatically added also to GetWired. But let's wait for next MySensors release to confirm it 100%.

In case your question was about supporting PJON's single wire communication - it would require designing a dedicated MCU Module and we are not planning to release it right now. If you know any good reason for us to consider it, please let us know :)

**Reply by szurr** [reply]
Usb gateway for me looks like rock solid. Only problems witch hardware such as relays ds or dht. Actually iam wondering about turn usb serial to Matt via node red plugin. 
One wire solution is most welcoming there many situations where you can simple hide one wire instead of some ethernet cable
